randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -895,14 +895,22 @@ Extra - room_id: [28]
   * L0 Hatch to Nightmare Hub West/Door to Flooded Access
   * Extra - door_idx: (63,)
   > Door to Zebesian Waters
-      Normal Damage ≥ 250 or Can Kill Tough Beam-Resistant Enemy
+      Any of the following:
+          Missiles ≥ 10 or Combat (Intermediate) or Can Use Power Bombs
+          Damage Boosts (Intermediate) and Normal Damage ≥ 250
+          Charge Beam and Plasma Beam
```

Comment:
what was the reasoning for requiring plasma beam for them?

Reply:
These aqua pirates are really fast and if you want to not get hit you'd need charge + 2 more beams to kill them in one shot, need to reevaluate this again tbh